packages/react-voting-app/src/pages/generate-passphrase/RegisterPage.tsx

```diff
@@ -134,23 +134,12 @@ function RegisterPage() {
                     <Backdrop className={classes.backdrop} open={loading}>
                         <CircularProgress color="inherit" />
                     </Backdrop>
-                    <Box className={classes.passphraseStats} component="div" m={1}>
-                        {
-                            passphrase.map(word => (
-                                <Chip
-                                    className={classes.chip} 
-                                    key={word}
-                                    label={word}
-                                />
-                            ))
-                        }
-                    </Box>
                 </CardContent>
               </div>
             </Card>
             <Fab 
                 onClick={handleFabClick}
-                disabled={passphrase.length < 12} 
+                disabled={passphrase.length === 0}
```

Comment:
it shouldn't base on the passphrase but on the completeness of data ... btw: I like the confirmation dialog